static/bidder-info/adverxo.yaml

```diff
@@ -0,0 +1,22 @@
+endpoint: "https://pbsadverxo.com/auction?id={{.AdUnit}}&auth={{.TokenID}}"
+endpointCompression: gzip
+maintainer:
+  email: "developer@adverxo.com"
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - native
+      - video
+  site:
+    mediaTypes:
+      - banner
+      - native
+      - video
+userSync:
+  iframe:
+    url: https://pbsadverxo.com/usync?type=iframe&gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
+    userMacro: $UID
+  redirect:
+    url: https://pbsadverxo.com/usync?type=image&gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
+    userMacro: $UID
```

Comment:
![image](https://github.com/user-attachments/assets/aa226797-b14f-4815-b49b-fdd9bb1f197f)
it doesnt work

Reply:
fixed